admin/admin-filters.php

```diff
@@ -90,16 +90,18 @@ public function update_check_locales( $locales ) {
 	}
 
 	/**
-	 * Adds a text direction dependent class to the body
+	 * Adds custom classes to the body
 	 *
-	 * @since 2.2
+	 * @since 2.2 Adds a text direction dependent class to the body.
+	 * @since 3.4 Adds a language dependent class to the body.
 	 *
 	 * @param string $classes Space-separated list of CSS classes.
 	 * @return string
 	 */
 	public function admin_body_class( $classes ) {
 		if ( ! empty( $this->curlang ) ) {
 			$classes .= ' pll-dir-' . ( $this->curlang->is_rtl ? 'rtl' : 'ltr' );
+			$classes .= ' post-language-' . ( $this->curlang->slug );
```

Comment:
The class will be added everywhere the current language is defined, so I suggest to remove `post`.
```suggestion
			$classes .= ' pll-lang-' . $this->curlang->slug;
```